.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,4 @@
+Before you submit a pull request please check that your code is functional and has as little bugs as possible. Then please list the following:
+
+- [ ] Functionality that has been added or removed
+- [ ] Issue that is being addressed (if there is one)
```

Comment:
Maybe also add:
`- [ ] Milestone that is being worked towards (if there is one that applies)` ?